github/repos.go

```diff
@@ -529,6 +531,12 @@ type RequiredStatusChecks struct {
 	Contexts *[]string `json:"contexts,omitempty"`
 }
 
+// RequiredPullRequestReviews represents the protection configuration for pull requests.
+type RequiredPullRequestReviews struct {
+	// Enforce pull request reviews for repository administrators.
+	IncludeAdmins *bool `json:"include_admins,omitempty"`
```

Comment:
@shurcooL :+1: I like that it exposes the requirements of the API in the type itself. I've made it so.

I also pushed 7634edb to backport this convention to `RequiredStatusChecks` because it also includes similarly required parameters like `include_admins`. I realize this is a breaking change, but I feel like it's a good idea to remain consistent within the structures that the `{Create,Update}BranchProtection` functions use. Furthermore, the GitHub HTTP API is itself still beta and open to breaking changes. If you think it's a bad idea, though, I can lop that commit off.